frontend/components/FoldableSwatches/components/Unfolded/index.jsx

```diff
@@ -204,7 +204,7 @@ const FoldableSwatchesUnfolded = ({
       {state => (
         <div className={styles.swatchesContainer}>
           {label && pdpSwatchesDisplayMode === 'headline' && pdpSwatchesPosition === 'variants' && (
-            <p className={styles.swatchHeadline} aria-hidden>{label}</p>
+            <p className={`${styles.swatchHeadline} swatches__headline`} aria-hidden>{label}</p>
```

Comment:
since at other lines in this components the `classnames` package is used to create classes, you should do this also here: `classnames(styles.swatchHeadline, 'swatches__headline')`